server/build/babel/plugins/transform-jsx-import.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function ({types}) {
```

Comment:
Add a comment with what this plugin does. May be we need a better name for the plugin. `transform` seems like too generic.